source/infrastructure/lib/aws-instance-scheduler-stack.ts

```diff
@@ -48,7 +48,7 @@ export interface AwsInstanceSchedulerStackProps extends cdk.StackProps {
 /*
 * AWS instance scheduler stack, utilizes two cdk constructs, aws-lambda-dynamodb and aws-events-rule-lambda.
 * The stack has three dynamoDB tables defined for storing the state, configuration and maintenance information.
-* The stack also includes one lambda, which is scheduled using a AWS CloudWatch Event Rule. 
+* The stack also includes one lambda, which is scheduled using an AWS CloudWatch Event Rule.
 * The stack also includes a cloudwatch log group for the entire solution, encrycption key, encyrption key alias and SNS topic,
```

Comment:
Wouldn't have noticed this without the change above, but encryption is spelled wrong here a couple of times. Not something to fix now but just something I noticed.